package.json

```diff
@@ -24,9 +24,13 @@
     "react-simple-chatbot": "^0.6.1",
     "styled-components": "^4.3.2"
   },
+  "devDependencies": {
+    "typescript": "^3.6.3"
+  },
   "scripts": {
-    "start": "yarn build-components && cd app && yarn start",
+    "start": "yarn build-component-types && cd app && yarn start",
     "build": "yarn build-components && cd app && yarn build",
+    "build-component-types": "cd components && tsc --EmitDeclarationOnly",
```

Comment:
If this does the trick do we need the full `build-components` command in `build` script?

Reply:
Yeah we could just use the `tsc` command and transpile everything to es5 with the types declarations into `build`.  Should just work for CRA consumers of components.

Reply:
so have `"build-components": cd components && tsc` and keep `build-component-types` as is.

Reply:
No, my mistake, I mixed up commands, build-components run something else :) In any case i think we can merge this.